### DISCUSSION TITLE
Codepen Example?

### DISCUSSION THREAD

**Discussion by GitHubProNinja** [creativelifeform/three-nebula#246] [General] [answered]
Hi,
This is an amazing library. Do you have one sample running on codepen.io?
The reason is that I am looking for an example without using npm, webpack, etc. I just want to see a code in plain JavaScript and references to necessary files only. Just like three.js samples that are readily available on codepen.io
It would be very helpful if you make one.
Thanks!

**Comment by rohan-deshpande** [comment, marked as answer]
Hi, sorry for taking so long to get to this. There is a codesandbox for getting started but you've requested just plain JS... sadly there's nothing like that. The examples in the repo used to be like this but I've recently changed those too. Anyway I'm posting the codesandbox just in case you've missed it - https://codesandbox.io/s/three-nebula-quickstart-kz6uv
